MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Tabs.xaml

```diff
@@ -0,0 +1,414 @@
+                                           VerticalAlignment="Stretch"
+                                           Mode="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Mode)}" 
+                                           wpf:ColorZoneAssist.Background="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Background)}"
+                                           wpf:ColorZoneAssist.Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Foreground)}">
+                                <Grid Background="{Binding ElementName=PART_ColorZone, Path=Background}">
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto"/>
+                                        <RowDefinition Height="1*"/>
+                                    </Grid.RowDefinitions>
+                                    <ContentPresenter Focusable="False" 
+                                                      Content="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:NavigationRailAssist.FloatingContent)}" />
+                                    <UniformGrid x:Name="HeaderPanel"
+                                         VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                                         Rows="1"
+                                         Grid.Row="1"
+                                         IsItemsHost="True"
+                                         Focusable="False"
+                                         HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"/>
+                                </Grid>
+                            </wpf:ColorZone>
+
+                        </Grid>
+                        <!--selected content-->
+                        <wpf:ColorZone x:Name="PART_ColorZoneSelectedContent" 
+                                       Padding="{TemplateBinding Padding}" 
+                                       VerticalAlignment="Stretch"
+                                       HorizontalAlignment="Stretch"
+                                       Mode="Standard" 
+                                       Foreground="{DynamicResource MaterialDesignBody}"
+                                       Background="{x:Null}">
+                            <ContentPresenter x:Name="PART_SelectedContentHost"
+                                              SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" 
+                                              Focusable="False"
+                                              Margin="{TemplateBinding Padding}"
+                                              TextElement.Foreground="{Binding ElementName=PART_ColorZoneSelectedContent, Path=Foreground}"
+                                              TextBlock.Foreground="{Binding ElementName=PART_ColorZoneSelectedContent, Path=Foreground}"
```

Comment:
I don't believe you need `TextBlock.Foreground` if `TextElement.Foreground` is set.